### DISCUSSION TITLE
我的电脑dll也不缺失去 搜索system32/里面也是有ws_32 这些dll全有 编译完成的dll依赖库都有 但是每次都是注入成功了 请求接口都没反应 19088也没有起来 版本是3.9.2.23 就是主分支main

### DISCUSSION THREAD

**Discussion by serviceqhd** [ttttupup/wxhelper#414] [Q&A]
![企业微信截图_20240510165140](https://github.com/ttttupup/wxhelper/assets/34151674/3c4fc0ec-57fb-4d36-87dd-7bfe29594200)
![企业微信截图_20240510165213](https://github.com/ttttupup/wxhelper/assets/34151674/1caf9aa9-fc27-4af7-9554-9d13adf08aee)

**Comment by serviceqhd** [comment]
![企业微信截图_20240510165832](https://github.com/ttttupup/wxhelper/assets/34151674/3d00b8a2-c7f2-43bf-8927-2bf84014bbc2)

**Comment by ttttupup** [comment]
可以直接在vs里调试一下注入程序，可能远程线程没有启动成功
